app/webpack.config.js

```diff
@@ -25,6 +25,7 @@ function makePortalConfig(name, urlPrefix) {
         name: name,
 
         entry: [
+        	"babel-regenerator-runtime",
```

Comment:
What does this do?

Also, could you please fix indentation here.

Reply:
this is module which allows to use async await functionality